source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -95,19 +95,6 @@ to learn about the principles of Ember's first edition in more detail.
 
 > Since the beginning, Ember has evolved alongside the JavaScript language. Ember has served as a testbed for features such as classes and modules, and for years has pushed more upstream than it has received back. With Octane, this tide has turned. Ember now feels as modern as any JavaScript framework, making the most of language features that it has helped shape. And its commitment to stable evolution means that this trend will continue for years to come. ~ _Dan Gebhardt [(@dgeb)](https://github.com/dgeb)_
 
-### Migrating Ember Apps
-
-Start migrating your existing Ember applications and benefit from Octane's interoperability with classic Ember with these useful reads:
-
-- [Rewriting Apps in Ember Octane](https://crunchingnumbers.live/2019/12/23/rewriting-apps-in-ember-octane/) by [Isaac Lee (@ijlee2)](https://github.com/ijlee2)
-- [The Ember Octane vs. Classic Ember Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)
```

Comment:
I think this one has also not been mentioned in the migration section yet - would it make sense to move it there with some prose, before deleting this section? What do you think?

Reply:
this one is here
https://github.com/ember-learn/ember-blog/pull/489/files#diff-9aebd6de851062be3f1b199a02c7df2aR107

row 107

Reply:
You're right, thank you for outlining!